syft/workers/base.py

```diff
@@ -259,8 +260,17 @@ def execute_command(self, message):
             tensor = getattr(_self, command)(*args, **kwargs)
         # Handle functions
         else:
+            # At this point, the command is ALWAYS a path to a
+            # function (i.e., torch.nn.functional.relu). Thus,
+            # we need to fetch this function and run it.
+
             sy.torch.command_guard(command, "torch_modules")
-            command = eval("self." + command)
+
+            paths = command.split(".")
+            command = self
+            for path in paths:
```

Comment:
I love this

Reply:
Thanks man! It took me longer than I care to admit to figure out. lol - but it worked out nicely!